src/main/java/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -41,6 +41,11 @@ public ResponseEntity<String> handleEntityNotFoundException() {
         return new ResponseEntity<>("Entity not found occurred", HttpStatus.INTERNAL_SERVER_ERROR);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<String> handleRegistrationException() {
+        return new ResponseEntity<>("", HttpStatus.CONFLICT);
```

Comment:
Let's make it more informative for the client
```suggestion
        Map<String, Object> body = new LinkedHashMap<>();
        body.put("timestamp", LocalDateTime.now());
        body.put("status", HttpStatus.CONFLICT);
        body.put("message", ex.getMessage());
        return new ResponseEntity<>(body, HttpStatus.CONFLICT);
```